### ISSUE TITLE
nginx fails to start with systemd after reboot

### ISSUE THREAD

**Issue by orenfromberg** [mempool/mempool#1452]
### Description

nginx would not start on boot and gave this error:
```
admin@n83bsaje632:~$ sudo systemctl status nginx

● nginx.service - A high performance web server and a reverse proxy server
     Loaded: loaded (/lib/systemd/system/nginx.service; enabled; vendor preset: enabled)
     Active: failed (Result: exit-code) since Thu 2022-03-24 23:22:58 EDT; 15min ago
       Docs: man:nginx(8)

Mar 24 23:22:49 n83bsaje632 systemd[1]: Starting A high performance web server and a reverse proxy server...
Mar 24 23:22:57 n83bsaje632 nginx[766]: nginx: [emerg] host not found in upstream "mempool.space" in /etc/nginx/snippets/nginx-mempool.conf:49
Mar 24 23:22:58 n83bsaje632 nginx[766]: nginx: configuration file /etc/nginx/nginx.conf test failed
Mar 24 23:22:58 n83bsaje632 systemd[1]: nginx.service: Control process exited, code=exited, status=1/FAILURE
Mar 24 23:22:58 n83bsaje632 systemd[1]: nginx.service: Failed with result 'exit-code'.
Mar 24 23:22:58 n83bsaje632 systemd[1]: Failed to start A high performance web server and a reverse proxy server.
```

#### Version

v2.3.1

### Steps to reproduce

run `sudo reboot now` and upon startup check `sudo systemctl status nginx`

### Expected behaviour

I expect nginx to be running

### Actual behaviour

nginx is not running

### Screenshots

n/a

#### Device or machine

Linux n83bsaje632 5.13.0-37-generic #42~20.04.1-Ubuntu SMP Tue Mar 15 15:44:28 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux
 Ubuntu 20.04.4 LTS

#### Additional info

n/a

**Comment by antonilol** [comment]
i also have this issue
i think it tries to resolve the domain name mempool.space before it has an internet connection
i worked around this with a delay
```ini
ExecStartPre=/usr/bin/sleep 30
```

**Comment by VajraOfIndra** [comment]
We also have this issue, see e.g. https://github.com/raspibolt/raspibolt/issues/919

**Comment by wiz** [comment]
Fixed by #1666
